apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/EventType.java

```diff
@@ -1,3 +1,18 @@
+/**
```

Comment:
Unrelated to your PR but this is what a full Maven build adds as this file was missing the license header so hopefully you don't object to it being added now ;)